lib/scss_lint/linter/nesting_depth.rb

```diff
@@ -30,8 +32,7 @@ def visit_rule(node)
 
     def ignore_selectors?(node)
       return unless config['ignore_parent_selectors']
-
-      simple_selectors(node.parsed_rules).all? do |selector|
+      simple_selectors(node.parsed_rules || null_selector).all? do |selector|
```

Comment:
Unless we intend to use the `null_selector elsewhere, it seems simpler (and less code) to have an early return.

```ruby
return unless node.parsed_rules
```